bosh-setup/README.md

```diff
@@ -4,11 +4,13 @@
     <img src="http://azuredeploy.net/deploybutton.png"/>
```

Comment:
Got it.
Another question: Can I put my own github username "bingosummer" as a contributor in the metadata file? Does it support the contributor tag besides the author?